### DISCUSSION TITLE
What is the relationship between Ingress-bfe and api-server?

### DISCUSSION THREAD

**Discussion by pinghe** [bfenetworks/ingress-bfe#24] [Q&A] [answered]
What is the relationship between Ingress-bfe and the three components just open-sourced?

**Comment by mileszhang2016** [comment, marked as answer]
ingress-bfe has no direct relation to the three compoments (i.e., API server, dashboard and conf-agent).
ingress-bfe (with [BFE server](https://github.com/baidu/bfe)) is used as ingress in Kubernetes environment.
API server, dashboard and conf-agent compose a complete control plane which can be used with BFE server as a layer-7 load balancer solution in Kubernetes, or in other environment.
